docs/why-xarray-beam.md

```diff
@@ -0,0 +1,75 @@
+# Why Xarray-Beam
+
+## Our goals
+
+Xarray-Beam is a Python library for building
+[Apache Beam](https://beam.apache.org/) pipelines with
+[Xarray](http://xarray.pydata.org/en/stable/) datasets.
+
+The project aims to facilitate data transformations and analysis on large-scale
+multi-dimensional labeled arrays, such as:
+
+-   Ad-hoc computation on Xarray data, by dividing a `xarray.Dataset` into many
+    smaller pieces ("chunks").
+-   Adjusting array chunks, using the
+    [Rechunker algorithm](https://rechunker.readthedocs.io/en/latest/algorithm.html)
+-   Ingesting large multi-dimensional array datasets into an analysis-ready,
+    cloud-optimized format, namely [Zarr](https://zarr.readthedocs.io/) (see
+    also [Pangeo Forge](https://github.com/pangeo-forge/pangeo-forge-recipes))
+-   Calculating statistics (e.g., "climatology") across distributed datasets
+    with arbitrary groups.
+
+## Our approach
+
+In Xarray-Beam, distributed Xarray datasets are represented by Beam PCollections
+of `(xarray_beam.Key, xarray.Dataset)` pairs and are manipulated by beam
+PTransforms, where {py:class}`~xarray_beam.Key` provides metadata specifying
+variables and chunk offsets for each "chunk" in the unordered PCollection.
```

Comment:
Kind of a long sentence. How about we end at "pairs" and start a new one?

Reply:
revised -- take another look?

Reply:
That reads better to me.